services/TokenCreate.proto

```diff
@@ -60,4 +60,5 @@ message TokenCreateTransactionBody {
     TokenType tokenType = 17; // IWA compatibility. Specifies the token type. Defaults to FUNGIBLE_COMMON
     TokenSupplyType supplyType = 18; // IWA compatibility. Specified the token supply type. Defaults to INFINITE
     int64 maxSupply = 19; // IWA Compatibility. Depends on TokenSupplyType. For tokens of type FUNGIBLE_COMMON - the maximum number of tokens that can be in circulation. For tokens of type NON_FUNGIBLE_UNIQUE - the maximum number of NFTs (serial numbers) that can be minted. This field can never be changed!
+    CustomFees customFees = 20; // Custom fee in Token Units/hbar that needs to be paid to the specified receiver on a TokenTransfer
```

Comment:
Based on this morning's discussion, there should be a new key for custom fees enhancement.